scripts/secrets-scan/run.sh

```diff
@@ -34,7 +32,7 @@ commits_file="$tmp_dir/commit_list.txt"
 gitleaks_config_container="${DOCKERREGISTRY}/typeform/gitleaks-config"
 gitleaks_container="zricethezav/gitleaks"
 gitleaks_version="v8.8.8"
-gitleaks_config_cmd="python gitleaks_config_generator.py --v8-config"
```

Comment:
This parameter (`--v8-config`) [was removed](https://github.com/Typeform/gitleaks-config/commit/a4363cb4a099425dd4e7d0fff858aaec84f6ec0d), so it should be fine.